migrating_from_ocp_3_to_4/planning-migration-3-4.adoc

```diff
@@ -73,6 +73,82 @@ For more information, see xref:../updating/updating-cluster-between-minor.adoc#u
+
+Some additional considerations for this option are:
+
+* The network device must terminate all secure TLS connections. Otherwise, if the connections are passed through to the OpenShift load balancer, the FQDN of the target application will be exposed to the client, and certificate errors will occur.
+
+* The applications must support this configuration, and they must not return links referencing the target cluster domain to the clients. If such links are leaked back to the client within the returned content, parts of the application may not load or work properly.
+
+==== Option 2: Set up the target cluster to accept the source DNS domain
+
+You can set up the target cluster to accept requests for migrated applications in the same DNS domain that was defined in the source cluster.
+
+* Non-secure (HTTP) access:
+
+  ** For the router in the default ingress controller of the target cluster to accept requests for applications in the source DNS domain (`app1.apps.ocp3.example.com`), you must create a route in the application’s project for the host name that was used in the source cluster:
++
+----
+$ oc expose svc app1-svc  --hostname app1.apps.ocp3.example.com \
+ -n app1-namespace
+----
++
+With this new route in place, any request for that FQDN will be accepted and sent to the corresponding application pods.
++
+When the application is migrated, a route is created in the target cluster domain (`app1.apps.ocp4.example.com`). The migrated application can be accessed using both of these host names.
+
+  ** You must create a specific DNS record for the FQDN defined in the route host name parameter that points to the IP address of the default load balancer of the target cluster. Specific DNS records take priority over wildcard records.
++
+The FQDN of the application will resolve to the load balancer of the target cluster. Requests for that FQDN will be accepted by the default ingress controller router because a route for that host name is exposed.
+
+* Secure TLS access:
++
+If the application needs a TLS/HTTPS connection, an additional step is required after you have created the route and DNS records:
++
+Replace the xref:../security/certificates/replacing-default-ingress-certificate.adoc#replacing-default-ingress[x509 certificate of the default ingress controller] created during the installation process with a custom certificate that includes the wildcard DNS domains for both the source and target clusters in the Subject Alternative Name (SAN) field. The new certificate will be valid for securing connections made using either of the two DNS domains.
++
+The certificate must be updated when it approaches its expiration date.
+
+==== Option 3: Deploy the target cluster in the source cluster domain
```

Comment:
Not required. This is an edge case and we have nothing meaningful to say about it.